.travis.yml

```diff
@@ -80,7 +80,7 @@ install:
   - phing recall -Dversion=$VERSION -Ddb.database=drupal -Ddb.user=lightning -Ddb.password=lightning -Ddb.host=127.0.0.1
   - cd docroot
   # Execute database updates, if there are any.
-  - drush updatedb --yes
+  - drupal update:execute
```

Comment:
At a minimum, we should update bullet 4 here: https://github.com/acquia/lightning/blob/8.x-2.x/UPDATE.md#tarball

A detailed description might not be needed, but we shouldn't give bad instructions.